referenceframe/primitives.go

```diff
@@ -59,6 +59,7 @@ func JointPositionsFromRadians(radians []float64) *pb.JointPositions {
 // InputEnabled is a standard interface for all things that interact with the frame system
 // This allows us to figure out where they currently are, and then move them.
 // Input units are always in meters or radians.
+// TODO: this really belongs in the motion service theres nothing stateful about the referenceframe package.
```

Comment:
@biotinker what do you think about this?  Should I ticket for backlog?

Reply:
We can't do that, as it would be a dependency cycle.

Reply:
Not sure I follow how.  Everything in this interface could be moved to the motion package I think.

Reply:
All frames operate on Inputs, so motion would not be able to import `referenceframe` anymore

Reply:
Actually I might be misunderstanding; are you suggesting migrating Inputs, or just the InputEnabled interface? The latter should be fine, IMO you should just do it this PR

Reply:
Yeah I was just suggesting the latter